APPSREPO-673 : Sync service included into ACS part II - ACS AWS Deplo…

constantinpopa10 opened the pull request:
…yment with EKS, AmazonMQ and Aurora

   - ACS chart version 2.1.0 which includes Sync service
   - use ACS image 6.1.1-RC1
   - configure the Sync chart to use the same AmazonMQ and AuroraMysql as ACS
   - upgraded to nginx 0.21.0 from 0.14.0, same as 'https://git.alfresco.com/Repository/acs-k8s-cluster/blob/master/scripts/deploy_acs_cluster.sh'
   - disable the nginx ingress provided by the alfresco-infrastructure as we install it separately beforehand
   - override the alfresco-sync-service.messaging.broker.endpoint with $MQ_ENDPOINT instead of providing separately the host, protocol and port
   - overwrite the messaging.broker.endpoint property via the EXTRA_JAVA_OPTS property

constantinpopa10 committed:
APPSREPO-673 : Sync service included into ACS part II - ACS AWS Deployment with EKS, AmazonMQ and Aurora
   - ACS chart version 2.1.0 which includes Sync service
   - use ACS image 6.1.1-RC1
   - configure the Sync chart to use the same AmazonMQ and AuroraMysql as ACS
   - upgraded to nginx 0.21.0 from 0.14.0, same as 'https://git.alfresco.com/Repository/acs-k8s-cluster/blob/master/scripts/deploy_acs_cluster.sh'
   - disable the nginx ingress provided by the alfresco-infrastructure as we install it separately beforehand
   - override the alfresco-sync-service.messaging.broker.endpoint with $MQ_ENDPOINT instead of providing separately the host, protocol and port
   - overwrite the messaging.broker.endpoint property via the EXTRA_JAVA_OPTS property

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Alfresco/acs-deployment-aws?pullRequest=72) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Alfresco/acs-deployment-aws?pullRequest=72) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Alfresco/acs-deployment-aws?pullRequest=72) it.</sub>

Commit:
APPSREPO-673

Changed the repo tag from 6.1.1-RC1 to 'feature-6.1.N-APPSREPO-673_include_sync_aws-6.1.1-SNAPSHOT' as the Sync service AMP was added to the AWS image after 6.1.1-RC1 was released
